Documentation/concepts/networking/masquerading.rst

```diff
@@ -0,0 +1,55 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _concepts_masquerading:
+
+Masquerading
+============
+
+IPv4 addresses used for pods are typically allocated from RFC1918 private
+address blocks and are thus not publicly routable. Cilium will automatically
+masquerade the source IP address of all traffic that is leaving the cluster to
+the IPv4 address of the node as the node's IP address is already routable on
+the network.
+
+.. image:: masquerade.png
+    :align: center
+
+This behavior can be disabled with the option ``masquerade: false`` in which
+case no masquerading will be performed.
+
+Configuration
+-------------
+
+Setting the routable CIDR
+  The default behavior is to exclude any destination within the IP allocation
+  CIDR of the local node. If the pod IPs are routable across a wider network,
+  that network can be specified with the option: ``native-routing-cidr:
+  10.0.0.0/8`` in which case all destinations within that CIDR will **not** be
+  masqueraded.
+
+Setting the masquerading interface (only available in iptables mode)
```

Comment:
Would it help to expand slightly on what "iptables mode" means here in this context?

Reply:
Ah, I see that it is explained a bit later below. I think it would be helpful to add a reference here. Something like:

```suggestion
Setting the masquerading interface (only available in iptables mode [see implementation modes below])
```